admin/mirage/config.js

```diff
@@ -64,4 +64,27 @@ export default function() {
     const organization = schema.organizations.findBy({ id: organizationId });
     return organization.update({ params });
   });
+
+  this.patch('/admin/users/:id', (schema, request) => {
+
+    const userId = request.params.id;
+
+    const userUpdated = {
+      'data': {
+        'type': 'users',
+        'id': userId,
+        'attributes': {
+          'first-name': 'johnUpdated',
+          'last-name': 'harryUpdated',
+          'email': 'johnharry@example.net',
+          'username': 'username123',
```

Comment:
Comme seul un utilisateur qui ne se connecte que par email peut être mis à jour, username sera toujours vide, alors autant faire de même ici

Reply:
Cette règle n'était pas implémentée encore. Je viens de mettre en place un mapping qui permet de mettre jour selon les params en entrée.

const userUpdated = {
      'data': {
        'type': 'users',
        'id': userId,
        'attributes': {
          'first-name': params.data.attributes['first-name'],
          'last-name': params.data.attributes['last-name'],
          'email': params.data.attributes['email'],
          'username': params.data.attributes['username'],
          'cgu': params.data.attributes['cgu'],
          'is-authenticated-from-gar': params.data.attributes['is-authenticated-from-gar'],
          'pix-orga-terms-of-service-accepted': params.data.attributes['pix-orga-terms-of-service-accepted'],
          'pix-certif-terms-of-service-accepted': params.data.attributes['pix-certif-terms-of-service-accepted']
        }
      }
    };
    return userUpdated;

La factory générer des données aléatoire du coup, on se retrouvait avec des valeurs aléatoires et des assert erronées.